tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist = py36, py37, py37-win
```

Comment:
This envlist is different from the `tox.yml` workflow file. Does that matter?

Reply:
actually that is not used when running the action so we could remove it